Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -465,6 +469,22 @@ extension BoringSSLRSAPublicKey {
         deinit {
             CCryptoBoringSSL_EVP_PKEY_free(self.pointer)
         }
+
+        fileprivate func getKeyPrimitives() -> (n: Data, e: Data) {
+            let key = CCryptoBoringSSL_EVP_PKEY_get0_RSA(self.pointer)
+
+            func getPrimitive(_ getPointer: (OpaquePointer?) -> UnsafePointer<BIGNUM>?) -> Data {
+                let ptr = getPointer(key)
+                let size = Int(CCryptoBoringSSL_BN_num_bytes(ptr))
+                var data = Data(count: size)
+                data.withUnsafeMutableBytes { dataPtr in
+                    _ = CCryptoBoringSSL_BN_bn2bin(ptr, dataPtr.baseAddress!.assumingMemoryBound(to: UInt8.self))
```

Comment:
I don't think we should need the assuming memory bound here, we should be safely able to pass the raw pointer through.